keras/engine/network.py

```diff
@@ -736,7 +736,6 @@ def run_internal_graph(self, inputs, masks=None):
                             else:
                                 output_masks = to_list(output_masks)
                             computed_tensors = [computed_tensor]
-                            computed_masks = [computed_mask]
```

Comment:
I think it is better to remain this line because it may be used afterward.

Reply:
Ok sure, I'll put a comment to say that the variable might be used in the future to avoid readers wondering what is going on.

Reply:
@gabrieldemarmiesse, I'm sorry to bother you, but the comment doesn't seem necessary.

Reply:
One may not understand why there is an unused variable in the codebase (and maybe do a PR to remove it again?).

Also I've read somewhere that comments should be used when some piece of code is not expected by the reader, to explain design decisions which are not obvious.

But you're the boss. If you still want me to remove the comment, I'll remove it.